### DISCUSSION TITLE
V5 如何注册一个带箭头的折线

### DISCUSSION THREAD

**Discussion by OShuxin** [antvis/G2#5931] [Q&A]
### 问题描述

如何注册一个带箭头的折线

### 平台

- 操作系统: [macOS, Windows, Linux, React Native ...]
- 网页浏览器: [Google Chrome, Safari, Firefox]

**Comment by pearmini** [comment]
参考自定义图形：https://g2.antv.antgroup.com/manual/extra-topics/customization#%E8%87%AA%E5%AE%9A%E4%B9%89%E5%BD%A2%E7%8A%B6shape
